subcommands/el2g/devices.go

```diff
@@ -28,6 +30,26 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 		Run:   doShow,
 	})
+
+	add := &cobra.Command{
+		Use:   "add <NC12 product-id> <device-id>",
+		Short: "Grant device access to the EdgeLock2GO",
```

Comment:
Thanks; this is _waaay_ more clear than a vague "add device to el2go"

Reply:
https://github.com/foundriesio/fioctl/pull/179/commits/472fd3aff3747f0ade09a0a476382dc63f084d22